README.md

```diff
@@ -1,54 +1,73 @@
 # Contraceptive Method Predictor
-- Author: Christopher Alexander
-- Contributors: Yik Hong Chan, Abhiket Gaurav, Valli A
+
+-   Author: Christopher Alexander
+-   Contributors: Yik Hong Chan, Abhiket Gaurav, Valli A
 
 A data analysis project to predict the contraceptive method choice of a woman based on her demographic and socio-economic characteristics.
+
 ## About
 
 In this project we are trying to answer the question: given the the demographic and socio-economic status are we able to predict the contraceptive method preferred by the individual? Answering this question is important because of the adverse effects contraceptive's can have on a person's health based on the usage. Furthermore, it also gives us an understanding of the factors that could have contributed to the reasons behind why the contraceptive market is such a big industry in the health care sector right now.
 
 The data set used in this project is a subset of the 1987 National Indonesia Contraceptive Prevalence Survey created by Tjen-Sien Lim. It was sourced from UCI Machine Learning Repository and can be found [here](http://archive.ics.uci.edu/ml/datasets/Contraceptive+Method+Choice). Each row in the dataset represents the demographics and social economic status of a woman in Indonesia (e.g., Age, Education, Religion, Standard of living index,etc.).
 
-To answer the predictive question posted above , we plan to build a predictive classification model. Before model building we will partition the data into training and test (70%:30%) and perform exploratory data analysis to understand various questions such as class imbalance, correlation, distribution of variables,etc. We will also find intital patterns we observe from the data which can help us in the model building process.
+To answer the predictive question posted above , we plan to build a predictive classification model. Before model building we will partition the data into training and test (70%:30%) and perform exploratory data analysis to understand various questions such as class imbalance, correlation, distribution of variables,etc. We will also find initial patterns we observe from the data which can help us in the model building process.
+
+Given there are a mix of continuous and categorical variables, and the outcome is a multi-class problem. We will start our approach by choosing a non linear model such as DecisionTree as our baseline as it handles mutli-class problems efficiently. Saying which for the initial stage of the project we cannot fix on a particular model until we try out different algorithms, other feature engineering steps required and hyper-parameter optimization. The levels of our target i.e `choice of contraceptive method` have been categorized into : 1=No-use, 2=Long-term, 3=Short-term. We will use F1 score as our evaluation metric as we want to give equal importance to all levels in the target.
```

Comment:
We will have to change the line about F1 score. We will say we are using accuracy and recall as we have categorized the target into 2 labels. As mentioned on slack.